src/Uno.UI/UI/Xaml/UIElement.Pointers.wasm.cs

```diff
@@ -295,14 +295,14 @@ private protected virtual void OnHitTestVisibilityChanged(HitTestVisibility oldV
 				// By default, elements have 'pointer-event' set to 'auto' (see Uno.UI.css .uno-uielement class).
 				// This means that they can be the target of hit-testing and will raise pointer events when interacted with.
 				// This is aligned with HitTestVisibilityProperty's default value of Visible.
-				SetStyle("pointer-events", "auto");
+				WindowManagerInterop.SetPointerEvents(HtmlId, true);
```

Comment:
Performance wise, it's not better to have 2 separates methods instead of a parameter?

Reply:
It may, but reading just one boolean won't change much for now.